### PR TITLE
Thunderbird for Android 8.0 release notes

### PR DESCRIPTION
**Pull request by kewisch**
@coreycb @cketti @madhens please review. Here is the rendered version:

Thunderbird:
```
===./app-thunderbird/src/main/res/raw/changelog_master.xml===
...
    <release version="8.0" versioncode="9999" date="2024-10-30">
        <change>Thunderbird for Android now looks like a Thunderbird!</change>
        <change>New folder drawer design.</change>
        <change>Migrate settings from K-9 Mail and Thunderbird Desktop within the app.</change>
        <change>Thunderbird is funded by users just like you. Contribute today, find the link in the settings.</change>
        <change>Simplified folder management by revising display and push classes.</change>
    </release>
...

===./app-metadata/net.thunderbird.android.beta/en-US/changelogs/9999.txt===
Thunderbird for Android version 8.0, based on K-9 Mail. Changes include:
- Thunderbird for Android now looks like a Thunderbird!
- New folder drawer design.
- Migrate settings from K-9 Mail and Thunderbird Desktop within the app.
- Thunderbird is funded by users just like you. Contribute today, find the link in the settings.
- Simplified folder management by revising display and push classes.

===github_notes===
New:
- Thunderbird for Android now looks like a Thunderbird!
- New Material 3 folder drawer gives Thunderbird a fresh new look.
- Migrate your settings from K-9 Mail to Thunderbird using an in-app settings import.
- Migrate your settings from Thunderbird Desktop to Thunderbird for Android using QR codes.
- You can make financial contributions via Google Play in-app purchases (play store variant), or linking to our fundraising page. Support Thunderbird today!
- Updated OAuth support for Fastmail and various other providers.
- Enhanced folder management, removed folder display and push classes.
- Updated translations for multiple languages, including Norsk Nynorsk and Irish.


Changed:
- Unread message colors updated for better visibility.
- A number of accessibility improvements when using Talkback.
- Addressed crashes in account management, crypto provider dialogs, and QR code import.


Unresolved:
- We are aware of user feedback from the beta list on the new folder drawer and seek to make improvements in upcoming releases.
```

K-9 Mail:
```
===./app-k9mail/src/main/res/raw/changelog_master.xml===
...
    <release version="8.0" versioncode="9999" date="2024-10-30">
        <change>New folder drawer design.</change>
        <change>Simplified folder management by revising display and push classes.</change>
    </release>
...

===./app-metadata/com.fsck.k9/en-US/changelogs/9999.txt===
- New folder drawer design.
- Simplified folder management by revising display and push classes.

===github_notes===
New:
- New Material 3 folder drawer gives Thunderbird a fresh new look.
- Updated OAuth support for Fastmail and various other providers.
- Enhanced folder management, removed folder display and push classes.
- Updated translations for multiple languages, including Norsk Nynorsk and Irish.


Changed:
- Unread message colors updated for better visibility.
- A number of accessibility improvements when using Talkback.
- Addressed crashes in account management, crypto provider dialogs, and QR code import.


Unresolved:
- We are aware of user feedback from the beta list on the new folder drawer and seek to make improvements in upcoming releases.
```

### CODE REVIEW

File: android_release/8.0.yml [hunk 1]
```diff
@@ -1,25 +1,67 @@
...
+    tag: new
+
+  - note: >
+      Migrate your settings from Thunderbird Desktop to Thunderbird for Android using QR codes.
+    thunderbird_only: true
+    tag: new
+
+  - note: >
+      You can make financial contributions via Google Play in-app purchases (play store
+      variant), or linking to our fundraising page. Support Thunderbird today!
```
Comment (cketti) [inline android_release/8.0.yml:35]: > … or linking to our fundraising page

We're doing the linking. Users can click/tap the link.

File: android_release/8.0.yml [hunk 2]
```diff
@@ -1,25 +1,67 @@
...
+  - note: Enhanced folder management, removed folder display and push classes.
+    short_note: Simplified folder management by revising display and push classes.
+    tag: new
+
+  - note: Updated translations for multiple languages, including Norsk Nynorsk and Irish.
+    tag: new
+
+
+  # Fixed
+  - note: Unread message colors updated for better visibility.
```
Comment (cketti) [inline android_release/8.0.yml:54]:
```suggestion
  - note: Updated text color of read messages in the message list for better visibility.
```

File: android_release/8.0.yml [hunk 3]
```diff
@@ -1,25 +1,67 @@
...
+
+  - note: Enhanced folder management, removed folder display and push classes.
+    short_note: Simplified folder management by revising display and push classes.
+    tag: new
+
+  - note: Updated translations for multiple languages, including Norsk Nynorsk and Irish.
+    tag: new
+
+
+  # Fixed
```
Comment (coreycb) [inline android_release/8.0.yml:53]: Looks like this should be
`# Changed`

Comment (coreycb) [reply android_release/8.0.yml:53]: Or
`# Fixed`

**Pull request merged**